### PR TITLE
V1.4 linkadd bug fixes

### LINKED ISSUE #197: [PE-D] listadd does not prompt user when command is incomplete
As opposed to taskadd, which does.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: aqcd/ped#6

### LINKED ISSUE #201: [PE-D] Mismatching example and feature used
Wrong command example used for the linkadd feature

-------------
Labels: `severity.High` `type.DocumentationBug`
original: RuiFengg/ped#3

### LINKED ISSUE #203: [PE-D] Invalid success message
Link added to the tag might be a link to some other resource and not a meeting link. Link added for a project module can be to the website textbook, difficult to check whether something is meeting link or not so maybe in the UG as well there's no need to say it to be a meeting link.

-------------
Labels: `severity.High` `type.FeatureFlaw`
original: RuiFengg/ped#5

### LINKED ISSUE #205: [PE-D] Example of command wrong
Found in Linkadd Feature in UG

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: tanwayne890/ped#1

### LINKED ISSUE #208: [PE-D] No error message
for command "linkadd 4"

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: tanwayne890/ped#4

### PR DESCRIPTION
**Pull request by JoeyToh**
Fix #201, #203, #197, #184

### CODE REVIEW

**Review by siqiang-ng — APPROVED** [review]
lgtm

File: docs/UserGuide.md [hunk 1]
```diff
@@ -88,6 +88,8 @@ Before you go on, here are some key terms and symbols you should know!
 #### <div> :bulb: </div>
 - The light bulb symbol will show you a tip to make your Projact experience smoother.
 
+#### <div> :exclamation: </div>
+- The exclamation mark symbol is a warning for any side effects that may result from the command, or any other important information you should know before you proceed.
```
Comment (yeo-kexin) [inline docs/UserGuide.md:92]: should we standardise all user input constraints to either information source or exclamation?

**Review by yeo-kexin — COMMENTED** [review]
minor thing to settle for UG, but otherwise lgtm!

File: docs/UserGuide.md (hunk 1 above)
Comment (siqiang-ng) [reply docs/UserGuide.md:92]: I was thinking the information source to be used for other purposes? Like additional information. But exclamation is like something that it is important to take note of at that point in time.

**Pull request merged**